iceoryx_platform/win/include/iceoryx_platform/platform_correction.hpp

```diff
@@ -35,13 +35,7 @@
 #pragma pointers_to_members(full_generality)
 #endif
 
-#undef CreateMutex
 #undef max
 #undef min
-#undef ERROR
 #undef interface
-#undef CreateSemaphore
-#undef NO_ERROR
 #undef OPEN_EXISTING
```

Comment:
With minimal effort, we could get rid of `interface` and `OPEN_EXISTING`. For `min`/`max`, there is an option to define `NOMINMAX` before including `windows.h`. What do you think, shall I tackle `interface` and `OPEN_EXISTING`?